integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesEvents.java

```diff
@@ -492,11 +503,258 @@ public void testDomainK8sEventsProcessingFailed() {
+
+    assertTrue(patchDomainResource(domainUid, domainNamespace1, new StringBuffer(patchStr)),
+        "patchDomainCustomResource failed");
+
+    domain1 = assertDoesNotThrow(() -> getDomainCustomResource(domainUid, domainNamespace1),
+        String.format("getDomainCustomResource failed with ApiException when tried to get domain %s in namespace %s",
+            domainUid, domainNamespace1));
+
+    //print out domain home in the new patched domain
+    logger.info("In the new patched domain domainHome is: {0}", domain1.getSpec().getDomainHome());
+    assertTrue(domain1.getSpec().getDomainHome().equals("/sharedpv/domains/" + domainUid),
+        "domainHome is not updated");
+
+    //print out mountPath in the new patched domain
+    logger.info("In the new patched domain mountPath is: {0}",
+        domain1.getSpec().getServerPod().volumeMounts().get(0).getMountPath());
+    assertTrue(domain1.getSpec().getServerPod().volumeMounts().get(0).getMountPath().equals("/sharedpv"),
+        "mountPath is not updated");
+
+    // verify the server pods are rolling restarted and back to ready state
+    logger.info("Verifying rolling restart occurred for domain {0} in namespace {1}",
+        domainUid, domainNamespace1);
+    assertTrue(verifyRollingRestartOccurred(podsWithTimeStamps, 1, domainNamespace1),
+        String.format("Rolling restart failed for domain %s in namespace %s", domainUid, domainNamespace1));
+
+    checkPodReadyAndServiceExists(adminServerPodName, domainUid, domainNamespace1);
+
+    for (int i = 1; i <= replicaCount; i++) {
+      logger.info("Checking managed server service {0} is created in namespace {1}",
+          managedServerPodNamePrefix + i, domainNamespace1);
+      checkPodReadyAndServiceExists(managedServerPodNamePrefix + i, domainUid, domainNamespace1);
+    }
+
+    logger.info("verify the domainHome,logHome and mountPath changes causes the domain roll events to be logged");
+    checkEvent(opNamespace, domainNamespace1, domainUid, DOMAIN_ROLL_STARTING, "Normal", timestamp);
+    checkEvent(opNamespace, domainNamespace1, domainUid, POD_CYCLE_STARTING, "Normal", timestamp);
+
+    CoreV1Event     event = getEvent(opNamespace, domainNamespace1,
+        domainUid, POD_CYCLE_STARTING, "Normal", timestamp);
+    logger.info(Yaml.dump(event));
+    assertTrue(event.getMessage().contains("/sharedpv"));
+    assertTrue(event.getMessage().contains("LOG_HOME"));
+    assertTrue(event.getMessage().contains("DOMAIN_HOME"));
+
+    event = getEvent(opNamespace, domainNamespace1,
+        domainUid, DOMAIN_ROLL_STARTING, "Normal", timestamp);
+    logger.info(Yaml.dump(event));
+    logger.info("verify the event message contains the domainHome changed message is logged");
+    assertTrue(event.getMessage().contains("domainHome"));
+    assertTrue(event.getMessage().contains("logHome"));
+    //********** I don't see the following message  **********************
```

Comment:
Is this due to pending issue OWLS-89879.   Add to the test description section.

Reply:
test is removed